Candidates.txt

```diff
@@ -1,3 +1,2 @@
 Bogdan Djuric bogdan-djuric
-Aleksandra your name goes here - replace second line with your name
-Ana Mucibabic Ana-Mucibabic
\ No newline at end of file
+Ana Mucibabic Ana-Mucibabic
```

Comment:
It's still not correct. You should (pull) have the latest master code on your local master branch in order to merge the latest master code with your branch

Reply:
Is it now correct? I pulled the latest master code on my local master branch.

Reply:
Now you have all changes, but you didn't merge it how it is expected. You should keep all three rows in that file and add your name on next free line